transcom/mymove
# INT B-18755 move history update weight pro gear expense

## [B-18755](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-18755)

## Summary

This is for fixing data displayed in Move History log when updates/uploads for PPM trip weight, pro-gear, and expense tickets occur when customer is submitting their documentation.

### How to test

1. Create a PPM move and get to the step where customer is able to upload PPM documents(trip weight, pro-gear, and expenses).
2. Create weights, pro-gears, and expense tickets.
3. Update any entry in those and check as a office user if all the necessary details are displayed in the Move History Log.

## Screenshots
![image](https://github.com/transcom/mymove/assets/146856854/57f149b9-b49c-4b08-a566-630fbafe02ad)
